Pipfile

```diff
@@ -8,7 +8,8 @@ verify_ssl = true
 [packages]
```

Comment:
Move Pipfile inside `devices` folder, each microservice have their own environment isolated.

Reply:
OK, I have made the changes